docs/reference/glossary.md

```diff
@@ -149,6 +149,35 @@ For more information, you can read [this blog post][udr-post].
 
 [udr-post]: https://authzed.com/blog/user-defined-roles
 
+## Wildcard (public)
```

Comment:
I don't think we should have such a concrete example in the glossary. I'd rather this just link to the schema reference for it. The schema reference can have two examples: one with users that describes the typical "public" usage and another to demonstrate that it can be used for more than just users.

Reply:
We have other schema examples in the glossary though, so if we wish to be consistent, we should remove those as well. That's why I placed one here

Reply:
We have a single for the actual definition of what a schema is. I'm not even sure wildcard should have an entry in the top-level glossary.

Reply:
I think it'll be important enough to warrant a definition, but I can remove the example if you like (I still feel we need a more complete example somewhere else though)